src/routes/web.php

```diff
@@ -1,23 +1,13 @@
 <?php
 
-use App\Http\Controllers\RegisterController;
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\LoginController;
 use App\Http\Controllers\UserController;
+use App\Http\Controllers\RegisterController;
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider and all of them will
-| be assigned to the "web" middleware group. Make something great!
-|
-*/
-
-
-Route::get('/', [LoginController::class, 'index'])->name('login');
+Route::get('/', [LoginController::class, 'index'])->name('login.auth');
+Route::post('/', [LoginController::class, 'auth']);
 Route::get('/register', [RegisterController::class, 'view'])->name('register');
 Route::post('/register', [RegisterController::class, 'register']);
-Route::get('/', [UserController::class, 'profile']);
+Route::get('/profile', [UserController::class, 'profile'])->name('user.profile');
+Route::post('/logout', [UserController::class, 'logout'])->name('logout');
```

Comment:
Follow the route naming convention!

https://webdevetc.com/blog/laravel-naming-conventions/

Reply:
Resolved this comment sir, Thankyou!